src/components/pages/Give/index.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+import { Member } from "../../../store/reducers/members";
+import { GiveForm } from "./GiveForm";
+import { Success } from "./Success";
+
+type Props = {};
+
+type State = {
+  operationIdentifer: string;
+  toMember?: Member;
+  amount?: string;
+  memo?: string;
+};
+
+/**
+ * Return a generally unique identifier.
+ */
+function getOperationIdentifer(): string {
+  return new Date().getTime().toString();
```

Comment:
good enough for this I guess lol. Would be a bit safer if you append some random nonce at the end probably